posts/2021-03-13-microprofile-config-2.0.adoc

```diff
@@ -0,0 +1,331 @@
+- <<ConfigValue, ConfigValue>>- an API class which holds a bunch of useful information about a specified config property
+- <<ConfigProperties, @ConfigProperties>>- an annotation which provides a way to retrieve a number of related config property values, with a shared specified prefix, into a CDI bean.
+- <<Property-Expression, Property Expression>>- a way of defining and expanding config property values inside of other config property values using the `${}` syntax.
+
+
+[#Prepare-For-Liftoff]
+== Prepare For Liftoff
+MicroProfile Config 2.0 is new to link:https://openliberty.io/downloads/#runtime_releases[Open Liberty 21.0.0.3] - so make sure you are using the latest version of Liberty. 
+
+Then, it's just 2 steps to launch:
+
+1. You’ll need to tell your build where all the awesome new features are defined. 
++
+The easiest way to do this it to point to the link:https://search.maven.org/artifact/org.eclipse.microprofile/microprofile/4.0.1/pom[MicroProfile 4.0 Repository] which includes the link:https://search.maven.org/artifact/org.eclipse.microprofile.config/microprofile-config-api/2.0/jar[MicroProfile Config 2.0 API].
++
+- If you’re using link:https://maven.apache.org/[Maven] add the following dependency to your pom.xml:
++
+.pom.xml
+[source,xml]
+----
+<dependency>
+	<groupId>org.eclipse.microprofile</groupId>
+	<artifactId>microprofile</artifactId>
+	<version>4.0.1</version>
+	<type>pom</type>
+	<scope>provided</scope> <!-- This package is built and released with Open Liberty -->
+</dependency>
+----
++
+- Or, if you're using link:https://gradle.org/[Gradle]:
++
+.build.gradle
+[source,gradle]
+----
+dependencies {
+    compileOnly group: 'org.eclipse.microprofile', name: 'microprofile', version: '4.0.1'
+}
+----
++
+
+1. You'll also need to enable the link:https://openliberty.io/docs/21.0.0.2/reference/feature/feature-overview.html[feature] in Open Liberty. 
++
+The easiest way to enable mpConfig-2.0 (MicroProfile Config 2.0’s feature name in Open Liberty) is to enable the microProfile-4.0 link:https://openliberty.io/docs/21.0.0.3/reference/feature/microProfile-4.0.html[convenience feature] (which enables mpConfig-2.0) in your server.xml:
++
+.server.xml
+[source,xml]
+----
+<featureManager>
+    <feature>microProfile-4.0</feature>
+</featureManager>
+----
++
+If you’d rather enable the individual feature instead, you’ll need to add `<feature>mpConfig-2.0</feature>` in addition to `<feature>cdi-2.0</feature>` which enables all the cool injection functionally.
+
+And just like that you’re ready for take-off!
+
+[#Getting-Up-To-Speed]
+== Getting Up To Speed
+If you’re a little unfamiliar with MicroProfile Config (no worries if you are), link:https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html[the spec] is a great place to start, or the link:https://openliberty.io/docs/21.0.0.3/external-configuration.html#[Open Liberty Docs]. 
```

Comment:
Might be better to point at one of the OL Guides here? The others are additional reference docs really.

Reply:
to put it another way, those reference docs are not a good place to "start"